sklearn/utils/validation.py

```diff
@@ -497,6 +497,13 @@ def indexable(*iterables):
     result : list of {ndarray, sparse matrix, dataframe} or None
         Returns a list containing indexable arrays (i.e. NumPy array,
         sparse matrix, or dataframe) or `None`.
+
+    Examples
+    --------
+    >>> from sklearn.utils import indexable
+    >>> a,b = [1, 2, 3],[2, 3, 4]
+    >>> indexable(a, b)
+    [[1, 2, 3], [2, 3, 4]]
```

Comment:
Please replace `b` by a numpy array, add a `c` value set to None and `d` set to scipy sparse array with 3 rows and 1 col.